free-2.md

```diff
@@ -58,12 +117,30 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+This project would require user reasearch determining the best way to create
+a DSL that is easy to grasp for users. Maybe the visual idea is not any easier
+than creating a typed PL in the classic sense.
+
+Once the langugae is created and an interpreter is built, there will be a fair
+amount of backend work to feed off Spotify&mdash;although they have a really
+useful and easy-to-learn API.
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+If the project involves making a typed DSL (typed as in using a keyboard not
+in the sense of datatypes) it will be less big of a project compared to having
+the goal of creating a visual language/program.
```

Comment:
I can imagine a pretty straightforward webapp; then you don't need all the complexity of most GUI toolkits